tests/cli/test_cli_preprocess.py

```diff
@@ -26,12 +26,15 @@ def test_preprocess_config_not_found(cli_runner):
 def test_preprocess_basic(cli_runner, config_path):
     """Test basic preprocessing with minimal config"""
     with patch("axolotl.cli.preprocess.do_cli") as mock_do_cli:
-        result = cli_runner.invoke(cli, ["preprocess", str(config_path)])
-        assert result.exit_code == 0
+        with patch("axolotl.cli.train.load_datasets") as mock_load_datasets:
```

Comment:
Does this need to be 
```suggestion
        with patch("axolotl.cli.preprocess.load_datasets") as mock_load_datasets:
```